benchmark/blobs_benchmark_visualiser.py

```diff
@@ -0,0 +1,90 @@
+# © Crown Copyright GCHQ
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Visualise the results of ``blobs_benchmark.py``."""
+
+import json
+import os
+
+
+# Function to print metrics table for each sample size
+def print_metrics_table(data: dict, sample_size: str) -> None:
+    """
+    Print a table for the given sample size with methods as rows and metrics as columns.
+
+    :param data: A dictionary where keys are sample sizes (as strings) and values are
```

Comment:
Edit this docstring to reflect the changes that no longer hard-code the methods or metrics